app/flatpages/policy_and_guidance.md

```diff
@@ -126,7 +126,7 @@ Public domain is not a valid license in many countries, please use CC0 or MIT in
 ## 5. Promotions and Advertisements (inc. asking for donations)
 
 You may note place any promotions or advertisements in any meta data including
```

Comment:
*not
*metadata

Reply:
I think I will fix those as well next week. Sorry I have not checked this until now. I have been extremely busy the past couple months.